package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "roosterjs",
-    "version": "7.3.0-darkmode.0",
+    "version": "7.3.2-darkmode.0",
```

Comment:
7.3.2 darkmode? just double checking.

Reply:
Yes. I made an emoji fix after 7.3.1, so this is 7.3.2